### ISSUE TITLE
CLI incompatible with yarn workspaces

### ISSUE THREAD

**Issue by SkyezerFox** [remotion-dev/remotion#805]
# Bug Report 🐛

When running `yarn start` on a project created with `yarn create video` inside a yarn workspace, the CLI launches, but crashes once the bundle has been built:

```
/Users/skyezer/Projects/my-cool-project/node_modules/@remotion/bundler/dist/get-package-manager.js:23
        throw new Error(`No lockfile was found in your project (one of ${paths
        ^

Error: No lockfile was found in your project (one of package-lock.json, yarn.lock, pnpm-lock.yaml). Install dependencies using your favorite manager!
    at getPackageManager (/Users/skyezer/Projects/my-cool-project/node_modules/@remotion/bundler/dist/get-package-manager.js:23:15)
    at Timeout._onTimeout (/Users/skyezer/Projects/my-cool-project/node_modules/@remotion/bundler/dist/update-available.js:30:77)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
```

I am running yarn workspaces with a root project, a frontend editor, and the video package.

#### Details

- OS: macOS 11.6.2
- Node: v16.16.1
- Remotion: remotion@2.6.5
- Yarn: v1.22.17

From the stack above, the culprit is [`get-package-manager.ts`](https://github.com/remotion-dev/remotion/blob/v2.6.5/packages/bundler/src/get-package-manager.ts) at tag `v2.6.5`. I've checked the `main` branch and this code has been modified to return "unknown" when it fails to find a package manager, but remains a problem in the latest release in NPM.

Would it be worth making this code recursive so it can correctly find the workspace root in multi-package projects? I'd be happy to do a PR for that, although I'm not aware of any side effects it might cause!

**Comment by SkyezerFox** [comment]
I have created a PR #806 that should fix this issue.

**Comment by SkyezerFox** [comment]
Fixed by #803

**Comment by JonnyBurger** [comment]
Fix is now released in 2.6.6. Thanks again 🤗
